index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <main>
+
+        <nav id="nav-bar">
+            <div class="left-nav-bar">
+                <img class="menu-logo" src="https://img.icons8.com/material-rounded/24/000000/menu.png" />
+                <img class="YouTube-logo" src="https://img.icons8.com/fluent/48/000000/youtube-play.png" />
+                <h3 class="title">YouTube</h3>
```

Comment:
I see you have used an image as a logo good job :clap:  However, instead of using just the play button as an image and the wording as just h3 tag use just one image for both.